src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const tabSelected = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': tabSelected.id === tab.id })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={tabSelected.id !== tab.id
+                  ? () => onTabSelected(tab)
+                  : () => {}
```

Comment:
It is better to put it in a separate function

Reply:
That's right. :)
![image](https://github.com/mate-academy/react_tabs-js/assets/102293862/20164cee-0e76-46ec-bfc3-0b61eaa9b607)